content/en/docs/Troubleshooting/_index.md

```diff
@@ -98,6 +98,7 @@ Use [Namespace selectors](/docs/installation/customization/#namespace-selectors)
 2. Which controller is experiencing the load? Each Kyverno controller has different responsibilities. See the [controller guide](/docs/high-availability/#controllers-in-kyverno) for more details. Each controller can be independently scaled, but before immediately scaling in any direction take the time to study the load.
 3. Are the default requests and limits still in effect? It is possible the amount of load Kyverno (any of its controllers) is experiencing is beyond the capabilities of the default requests and limits. These defaults have been selected based on a good mix of real-world usage and feedback but **may not suit everyone**. In extremely large and active clusters, from Kyverno's perspective, you may need to increase these.
 4. What do your monitoring metrics say? Kyverno is a critical piece of cluster infrastructure and must be monitored effectively just like other pieces. There are several metrics which give a sense of how active Kyverno is, the most important being [admission request count](/docs/monitoring/admission-requests/). Others include consumed memory and CPU utilization. Sizing should always be done based on peak consumption and not averages.
+5. Have you checked the number of pending update or generate requests? In addition to the admission request count metric, you can use `kubectl -n kyverno get updaterequests` or `kubectl -n kyverno get generaterequests` to get a sense of the request count. If there are a lot of Pending requests this could be a sign of a permissions issue - are you granting the controller clusterrroles permissions to the resources and operations it needs?
```

Comment:
```suggestion
5. Have you checked the number of pending update requests when using generate or mutate existing rules? In addition to the admission request count metric, you can use `kubectl -n kyverno get updaterequests` to get a sense of the request count. If there are many requests in a `Pending` status this could be a sign of a permissions issue or, for clone-type generate rules with synchronization enabled, excessive updates to the source resource. Ensure you grant the background controller the required permissions to the resources and operations it needs, and ensure Kyverno is able to label clone sources.
```